tests/e2e/lib/container

```diff
@@ -103,12 +102,10 @@ create_qm_node() {
         echo "RUN dnf install qm -y &> /dev/null" >> ContainerFile.node"${nodeID}"
 
         # Execute qm setup
+        # Use --skip-systemctl true in podman build, systemd is not running
+        # Do not set prameter once running on running container with systemd running
         echo "RUN /usr/share/qm/setup --skip-systemctl true 2>&1 > /tmp/qm-setup.log || echo "QM setup failed, please check /tmp/qm-setup.log."" >> ContainerFile.node"${nodeID}"
 
-        # HACK: add /dev/fuse to qm.container. It will allow a container set limits inside
-        # another container.
-        echo 'RUN echo "AddDevice=-/dev/fuse" >> /usr/share/containers/systemd/qm.container' >> ContainerFile.node"${nodeID}"
-
```

Comment:
or maybe this broke the CI?

Reply:
I thin k yes

Reply:
Will run it manually, first

Reply:
I think the problem is not this but the other thing I pointed but we never know.

Reply:
Ack